src/main/scala/SbtFrege.scala

```diff
@@ -44,21 +44,21 @@ object SbtFregec extends AutoPlugin {
   }
 
   def scopedSettings(scope: Configuration, dirName: String) = Seq(
-    fregeSource in scope := (sourceDirectory in scope).value / "frege",
-    fregeTarget in scope := baseDirectory.value / "target" / dirName,
-    sourceGenerators in scope += Def.task {
+    scope / fregeSource := (scope / sourceDirectory).value / "frege",
```

Comment:
method in in trait ScopingSetting is deprecated (since 1.5.0): `in` is deprecated; migrate to slash syntax - https://www.scala-sbt.org/1.x/docs/Migrating-from-sbt-013x.html#slash